server/etcdserver/api/v3rpc/grpc.go

```diff
@@ -16,11 +16,10 @@ package v3rpc
 
 import (
 	"crypto/tls"
-	"math"
-
 	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
 	"go.etcd.io/etcd/client/v3/credentials"
 	"go.etcd.io/etcd/server/v3/etcdserver"
+	"math"
```

Comment:
Nitpick: I belive the previous ordering was correct.

Reply:
this is something done by Goland automatically I think. I just changed it back.